woocommerce/woocommerce-ios
# Updated Charts pod from 3.3.0 to 3.6.0, that includes Xcode 12 support

## Description
As part of our Xcode 12 migration, we should update our Pod libraries, because some of them show warnings about the `IPHONEOS_DEPLOYMENT_TARGET` set to iOS 8.0, that is no more supported.
In this PR I updated the Charts pod from v.3.3.0 to v.3.6.0 that now supports Xcode 12.

## Testing
- Check that the Stats in the dashboard are working like before.
- Make sure that this warning disappear:
<img width="502" alt="Schermata 2020-10-16 alle 11 26 44" src="https://user-images.githubusercontent.com/495617/96241989-3b112f80-0fa3-11eb-9da8-ce87fc338d34.png">


Update release notes:
